### PR TITLE
Statement that cache can be enabled|disabled only from command line starting Magento 2.2 is false #4846

### PR DESCRIPTION
**Pull request by kirtinariya1**
## Purpose of this pull request

This pull request (PR) will be merge the show correct 

## Affected DevDocs pages
- https://devdocs.magento.com/guides/v2.2/config-guide/cli/config-cli-subcommands-cache.html
- https://devdocs.magento.com/guides/v2.3/config-guide/cli/config-cli-subcommands-cache.html

## Links to Magento source code
N/A

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by hostep** [comment]
@kirtinariya1: the statement is true. But only when Magento runs in production mode. In that case you can't enable or disable the caches using the backend. However, if Magento runs in developer mode, you can still change them using the backend.

So you'd best rephrase the docs a bit, since it wasn't really clear it is only the case for production mode.

**Comment by kirtinariya1** [comment]
@hostep Thanks for your attentions guide, I pleasure for that. I correct the thing, you can enable or disable cache types only using the command line, while running Magento in production mode. If Magento running in developer mode then you can enable or disable cache types using command line or manually.

File: guides/v2.2/config-guide/cli/config-cli-subcommands-cache.md [hunk 1]
```diff
@@ -127,15 +127,14 @@ config_integration_api: 1
...
 
 {:.bs-callout .bs-callout-info}
-Starting in version 2.2, you can enable or disable cache types _only_ using the command line. Before doing so, you must manually make `<magento_root>/app/etc/env.php` writeable by the [Magento file system owner]({{ page.baseurl }}/install-gde/prereq/file-system-perms.html). You can clean (also referred to as _flush_ or _refresh_) cache types using either the command line or the [Magento Admin](https://glossary.magento.com/magento-admin).
+Starting in version 2.2, you can enable or disable cache types only using the command line while running Magento in production mode. Also when Magento runing in developer mode, then you can enable or disable cache types using command line or manually. Before doing so, you must manually make `<magento_root>/app/etc/env.php` writeable by the [Magento file system owner]({{ page.baseurl }}/install-gde/prereq/file-system-perms.html). You can clean (also referred to as _flush_ or _refresh_) cache types using either the command line or the [Magento Admin](https://glossary.magento.com/magento-admin).
```
Comment (erikmarr) [inline guides/v2.2/config-guide/cli/config-cli-subcommands-cache.md]:
```suggestion
Starting in version 2.2, you can only enable or disable cache types using the command line while running Magento in production mode. If running Magento in developer mode, you can enable or disable cache types using the command line or manually. Before doing so, you must manually make `<magento_root>/app/etc/env.php` writeable by the [Magento file system owner]({{ page.baseurl }}/install-gde/prereq/file-system-perms.html). You can clean (also referred to as _flush_ or _refresh_) cache types using either the command line or the [Magento Admin](https://glossary.magento.com/magento-admin).
```

File: guides/v2.3/config-guide/cli/config-cli-subcommands-cache.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ config_integration_api: 1
 This command enables you to enable or disable all cache types or only the ones you specify. Disabling cache types is useful during development because you see the results of your changes without having to flush the cache; however, disabling cache types has an adverse effect on performance.
 
 {:.bs-callout .bs-callout-info}
-Starting in version 2.2, you can enable or disable cache types _only_ using the command line. Before doing so, you must manually make `<magento_root>/app/etc/env.php` writeable by the [Magento file system owner]({{ page.baseurl }}/install-gde/prereq/file-system-perms.html).
+Starting in version 2.2, you can enable or disable cache types only using the command line while running Magento in production mode. Also when Magento runing in developer mode, then you can enable or disable cache types using command line or manually. Before doing so, you must manually make `<magento_root>/app/etc/env.php` writeable by the [Magento file system owner]({{ page.baseurl }}/install-gde/prereq/file-system-perms.html).
```
Comment (erikmarr) [inline guides/v2.3/config-guide/cli/config-cli-subcommands-cache.md]:
```suggestion
Starting in version 2.2, you can only enable or disable cache types using the command line while running Magento in production mode. If running Magento in developer mode, you can enable or disable cache types using the command line or manually. Before doing so, you must manually make `<magento_root>/app/etc/env.php` writeable by the [Magento file system owner]({{ page.baseurl }}/install-gde/prereq/file-system-perms.html).
```

**Comment by erikmarr** [comment]
@kirtinariya1 please update your PR description to include:
guides/v2.2/extension-dev-guide/events-and-observers.md 
in the affected devdocs section.

File: guides/v2.3/config-guide/cli/config-cli-subcommands-cache.md (hunk 2 above)
Comment (kirtinariya1) [reply guides/v2.3/config-guide/cli/config-cli-subcommands-cache.md]: @erikmarr Content has updated.

**Comment by erikmarr** [comment]
Hi @kirtinariya1 it doesn't look like you committed the changes just yet.

**Comment by erikmarr** [comment]
Hi @kirtinariya1 thanks for making the changes. I noticed that changes to the ..\extension-dev-guide/events-and-observers.md topic are no longer present in this PR. Is that intentional?

**Comment by kirtinariya1** [comment]
@erikmarr This PR for cache updation. You mentioned event-and-observers that was different issues Nd by mistake I was committed to this same PR so that's why I have reverted that change. Now this PR is ready and update to correct.

**Comment by erikmarr** [comment]
running tests

**Comment by erikmarr** [comment]
Thanks @kirtinariya1!

**Pull request merged**